build/ApphostTemplate.proj

```diff
@@ -0,0 +1,49 @@
+<Project ToolsVersion="15.0" DefaultTargets="ExtractAppHostToOutput">
```

Comment:
This file should be `AppHostTemplate.proj` (cause of your case-sensitive file system build errors).